### PR TITLE
spec: fix env var checking and secondary project

### PR DESCRIPTION
**Pull request by tbpg**

### CODE REVIEW

File: spec/kokoro-run-all.sh [hunk 1]
```diff
@@ -19,14 +19,8 @@ source $KOKORO_GFILE_DIR/secrets.sh
...
-  GOOGLE_CLOUD_PROJECT
-  GOOGLE_APPLICATION_CREDENTIALS
-  GOOGLE_CLOUD_STORAGE_BUCKET
-  ALTERNATE_GOOGLE_CLOUD_STORAGE_BUCKET
-  GOOGLE_CLOUD_PROJECT_SECONDARY
-  GOOGLE_APPLICATION_CREDENTIALS_SECONDARY
-  GOOGLE_CLOUD_KMS_KEY_NAME
-  GOOGLE_CLOUD_KMS_KEY_RING
+  GOOGLE_CLOUD_KMS_KEY_NAME \
+  GOOGLE_CLOUD_KMS_KEY_RING \
```
Comment (dazuma) [inline spec/kokoro-run-all.sh]: It needs to be like this:

```
for REQUIRED_VARIABLE in \
  GOOGLE_CLOUD_KMS_KEY_NAME \
  GOOGLE_CLOUD_KMS_KEY_RING
do
```

Comment (tbpg) [reply spec/kokoro-run-all.sh]: Whoops! Done. Thanks!

File: spec/kokoro-run-all.sh [hunk 2]
```diff
@@ -19,13 +19,7 @@ source $KOKORO_GFILE_DIR/secrets.sh
 export BUNDLE_GEMFILE=
 
 for REQUIRED_VARIABLE in
```
Comment (dazuma) [inline spec/kokoro-run-all.sh]: We still need a backslash after ```for REQUIRED_VARIABLE in```

Comment (dazuma) [reply spec/kokoro-run-all.sh]: (bash is annoying)

Comment (tbpg) [reply spec/kokoro-run-all.sh]: Doh. Done. Thanks!

**Pull request merged**